dist/angular-chart.js

```diff
@@ -311,15 +311,18 @@
       var dataset = {
         labels: labels,
         datasets: [{
-          data: data,
-          backgroundColor: colors.map(function (color) {
-            return color.pointBackgroundColor;
-          }),
-          hoverBackgroundColor: colors.map(function (color) {
-            return color.backgroundColor;
-          })
+          data: data
         }]
       };
+      colors.map(function (color) {
```

Comment:
[map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) returns a newly created array which is not needed here, so use `foreach` instead.